File: Leetcode/Leetcode-DSA-course/Hashing/3242-count-elements-with-maximum-frequency/count-elements-with-maximum-frequency.java
class Solution {
    public int maxFrequencyElements(int[] nums) {
        int freq[] = new int[101];
        Arrays.fill(freq,0);
        int max_freq = 1;
        for (int num : nums) {
            freq[num]++;
            max_freq = Math.max(max_freq, freq[num]);
        }
        int sum = 0;
        for (int i = 1; i < 101; i++) {
            if (freq[i] == max_freq)
                sum += freq[i];
        }
        return sum;
    }
}

File: Leetcode/Leetcode-DSA-course/Hashing/3242-count-elements-with-maximum-frequency/count-elements-with-maximum-frequency.ts
function maxFrequencyElements(nums: number[]): number {
        let freq = new Array(101).fill(0);
        let max_freq = 1;
        for (let num of nums) {
            freq[num]++;
            max_freq = Math.max(max_freq, freq[num]);
        }
        let sum = 0;
        for (let i = 1; i < 101; i++) {
            if (freq[i] == max_freq)
                sum += freq[i];
        }
        return sum;
};